fastapi_validation_i18n/translator.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, locale: str, locale_path: str = 'locale'):
     def load_translation(self, file_key: str) -> Any | Dict[str, Any] | None:
         file_path = f'{self.locale_path}/{self.locale}/{file_key}.json'
```

Comment:
**Suggestion:** Use pathlib for path manipulations instead of string concatenation for better readability and simplicity.
```suggestion
        from pathlib import Path
        file_path = Path(self.locale_path) / self.locale / f'{file_key}.json'
```